modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/AuditBarChart.js

```diff
@@ -111,4 +113,5 @@ function CustomTick(props) {
 AuditBarChart.propTypes = {
 	bars: PropTypes.array.isRequired,
 	data: PropTypes.array.isRequired,
+	langDir: PropTypes.string.isRequired,
```

Comment:
Would it make sense to you if we just had `rtl` as a boolean prop and no `langDir` prop? This way we wouldn't have to do any repetitive string comparisons. Only reason I could see us keeping it as a string, is if we anticipate more values than just `ltr` and `rtl`. If its just those two, we can assume `ltr` and make the prop `rtl`.

Reply:
Yes, it makes sense. We can hard-coded, instead of a string prop, a boolean prop named `rtl`. I'll change it :)